org-cyf-itd/content/blocks/step-5/instructions/index.md

```diff
@@ -1,28 +1,88 @@
 
-### Take the test
+In this exercise, you must build your own homepage on CodePen that meets some specific requirements. Give it your own personal touch. We know that people who do well on our courses show **creativity**, **curiosity**, **persistence**, and **rigour**. That’s what we’re looking for from you in this project.
 
-1. Go to the [Duolingo Practice Test](https://englishtest.duolingo.com/test/practice) page
-1. Take the test, which will be around 45 minutes
-1. Take a screenshot of your result
-1. Upload your image to this step in the Learning Journey platform
+{{<note type="tip" title="Tip">}}
+- Make good use of web search and AI
+{{</note>}}
 
-{{<blocklink
-  src="https://blocks.codeyourfuture.io/#introduction"
-  name="CYF Blocks"
-  caption="Code Your Future">}}
+### What Should You Do?
+
+##### 1. Make a free account on CodePen
+- Sign up for a free account on [CodePen](https://codepen.io/).
+
+##### 2. Fork the CYF starter page
+
+1. Fork (make a copy of) the CYF starter page, [Personal Home Page Template](https://codepen.io/IntroToTech/pen/PomeEod), to your own CodePen account.
+    - Here are the [instructions to fork a CodePen page](https://blog.codepen.io/documentation/forks/).
+    - Make sure you're logged into CodePen when you fork the page so that your work is saved.
+ 
+{{<note type="note" title="Note">}}
+- The CYF template is set up to use JavaScript to check whether your HTML code passes 10 automated tests.
+- You must not modify the JavaScript code.
+{{</note>}}
+
+##### 3. Prepare your homepage
+
+1. Prepare your homepage in the page you forked.
+2. Write your HTML code in the **HTML** tab and your CSS code in the **CSS** tab.
+    - You may delete the HTML comment in the **HTML** tab.
+3. Your homepage must meet the following criteria, which are written in a special format called _Given/When/Then_.
+
+{{<note type="note" title="Given/When/Then Criteria">}}
+**Given a personal homepage**
+
+**When** I look in the test console  
+_Then_ all 10 tests are passing
+
+**When** I read the content  
+_Then_ I learn something about you
+
+**When** I view the page  
+_Then_ the design is unique and the images are displayed properly
+
+**When** I play the video on the page
+_Then_ the video can play normally
+
+**When** I validate the HTML
+_Then_ the HTML is valid
+{{</note>}}
+
+
+##### 4. Validate your code
+
+1. Use CodePen’s "Analyze HTML" feature to help you find errors in your HTML code. Then fix the errors yourself manually.
+
+![HTML Analyzer](codepen-html-analyzer.png)
+
+2. Use a similar approach to identify and fix CSS errors.
+
+
+##### 5. Submit your CodePen page link  
+
+- Submit the link to your CodePen page on the [CYF Course Platform](https://application-process.codeyourfuture.io/).
+
+> [!TIP]
+> Your work is not judged on aesthetics in this project. We are interested in code correctness and specification compliance.
+
+<!--
+### Advice
```

Comment:
I cannot see this on the preview. Should I ignore it and approve it anyway, CJ, and we can check in prod later? @cjyuan